.ratignore

```diff
@@ -39,6 +39,7 @@ site/_layouts/news.html
 site/_layouts/news_item.html
 site/_layouts/page.html
 site/_sass/**
+site/Gemfile.lock
```

Comment:
I believe this change has been in master branch?

Reply:
Sure, I've removed it.